frontend/packages/policy-editor/src/utils/index.ts

```diff
@@ -115,23 +115,22 @@ export const mapSubjectTitleToSubjectString = (
   const subject: PolicySubject = subjectOptions.find(
     (s) => s.subjectTitle.toLowerCase() === subjectTitle.toLowerCase(),
   );
-  return `urn:${subject.subjectSource}:${subject.subjectId}`;
+  if (subject === undefined) return;
+  return `urn:${subject?.subjectSource}:${subject?.subjectId}`;
```

Comment:
I recommend transitioning to `!subject`, as it will account for both `null` and `undefined`. This adjustment allows us to remove the null checks in the next line. 😊 

```suggestion
  if (!subject) return;
  return `urn:${subject.subjectSource}:${subject.subjectId}`;
```

Reply:
The reason I had all these checks was to avoid getting the error on my own machine when I tried to remove the DAGL and REGNA roles that was added automatically. 
Are you able to open studio locally on your machine on this branch and check that it works as expected when removing DAGL and REGNA now after I have updated the code? 😄